oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/CoreModule.java

```diff
@@ -48,6 +49,9 @@ public CoreModule() {
         classes.add(DownsamplingConfigService.class);
         classes.add(IComponentLibraryCatalogService.class);
 
+        classes.add(IWorkerInstanceGetter.class);
+        classes.add(IWorkerInstanceSetter.class);
```

Comment:
How about call these two as WorkerInstanceManager? Setter/Getter looks too abstract as a service to me.

Reply:
Anything can called manager, but managed what? Interface represent a behavior, so I named the class by his explicit behavior.

Reply:
I have no preference here, just asking, because we are using ModuleManager, etc.